tests/user/actions/set_password_temporary.yml

```diff
@@ -0,0 +1,30 @@
+---
+action: user.set_password_temporary
+user_id: 1
+db:
+  user/2/meeting_id: 1
+
+cases:
+- name: superadmin
+  db:
+    user/1/organisation_management_level: superadmin
+  is_allowed: true
+
+- name: orga manager
+  db:
+    user/1/organisation_management_level: can_manage_organisation
+  is_allowed: true
+
+- name: orga user manager
+  db:
+    user/1/organisation_management_level: can_manage_users
+  is_allowed: true
+
+- name: meeting manager
+  permission: user.can_manage
+  payload:
+    id: 2
+  db:
+    user/2/group_$_ids: ["1"]
+
+  is_allowed: true
```

Comment:
Please add a test for a non-temporary user. I wonder what your response is since this is business logic executed by the backend, but after the permission check. Two cases:
- you disallow it, no backend action
- you allow it -> backend will disallow it.

I think the permission service should allow it: It is not the part of checking who to set the password, but just of "is the request user allowed to do it". So the request user can do it (if the user is valid, but this is checked in the backend)

Reply:
Funny fact. The permission service uses the same code for `user.set_password` and `user.set_password_temporary`

https://github.com/OpenSlides/openslides-permission-service/pull/164/files#diff-c2c4eec56d899f0952451def90200bbb7bab6d6e10d71bc3b669a9688d1c1ca6R24-R25

You can find the test for user.set_password here `https://github.com/OpenSlides/openslides-permission-service/blob/d0600fdab16b6b20d9eac242ef5efdef7a39a999/tests/user/actions/set_password.yml`

If a manager calls set_password_temporary on a non-temporary user, the permission service disallows it. I would like to keep it this way so it is possible to call the generic function.